src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/faktavurderinger/Vurdering.kt

```diff
@@ -1,10 +1,18 @@
 package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.faktavurderinger
 
+import java.time.LocalDateTime
+
 sealed interface Vurdering {
     val svar: SvarJaNei?
     val resultat: ResultatDelvilkårperiode
 }
 
+sealed interface AutomatiskVurdering : Vurdering {
```

Comment:
Syns du dette ser fornuftig ut, @blommish ?
@

Reply:
* `inputfakta` trenger vi å lagre denne på selve vurderingen? Vi har allerede informasjonen i grunnlaget som er fakta
* `gitHash` burde denne legges på høyere nivå? På `GeneriskVilkårperiode` (denne kan ev gjøres senere - den trenger nog litt håndtering)
* `tidspunktForVurdering`, er ikke dette tidspunktet hele `GeneriskVilkårperiode` ble vurdert?

Reply:
* `tidspunktForVurdering` blir samme tid som `endret_tid` for GeneriskVilkårperiode, så du har rett i at vi ikke trenger å lagre den to ganger.

* `inputfakta` trenger muligens å inneholde fødselsdagen til brukeren, siden denne blir brukt når man gjør vurderingen. Dette i tilfelle vi gjør vurderingen basert på feil fødselsdato. Fødselsdato kommer riktignok fra grunndata, så det er kanskje ikke nødvendig å ta hensyn til at denne kan være feil?

* Tenker du `gitHash` burde legges på `GeneriskVilkårperiode`? Den er i utgangspunktet bare påkrevd dersom det er en "Automatisk Vurdering"

Reply:
* `inputfakta` - Vi kan sikkert ta en prat ang `inputfakta` med teamet og se hva vi tenker er nødvendig her
* `githash` ja, jeg tenker det er enklere å bare legge på den på alle. Då får man også informasjon om hvilken kjørende kode som vært i bruk npr man lagrer/endrer en vilkårperiode

Reply:
@blommish Dersom `githash` skal legges direkte på `Vurdering`, så tenker jeg at jeg fjerner `githash` fra denne PR, og heller legger det inn i en egen PR senere

Reply:
Fjerner `githash` fra denne PR:  21c3cacad793ae51b878cd68f8a75b527797e1b3